www/js/metrics/CarbonFootprintCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState, useMemo } from 'react';
+import { View } from 'react-native';
+import { Card, Text, useTheme} from 'react-native-paper';
+import { MetricsData } from './metricsTypes';
+import { cardStyles } from './MetricsTab';
+import { formatDateRangeOfDays, parseDataFromMetrics, generateSummaryFromData, calculatePercentChange, segmentDaysByWeeks, isCustomLabels } from './metricsHelper';
+import { useTranslation } from 'react-i18next';
+import BarChart from '../components/BarChart';
+import { getAngularService } from '../angular-react-helper';
+import ChangeIndicator from './ChangeIndicator';
+import color from "color";
+
+type Props = { userMetrics: MetricsData, aggMetrics: MetricsData }
+const CarbonFootprintCard = ({ userMetrics, aggMetrics }: Props) => {
+    const FootprintHelper = getAngularService("FootprintHelper");
+    const { colors } = useTheme();
+    const { t } = useTranslation();
+
+    const [emissionsChange, setEmissionsChange] = useState({});
+ 
+    const userCarbonRecords = useMemo(() => {
+        if(userMetrics?.distance?.length > 0) {
+            //separate data into weeks
+            const [thisWeekDistance, lastWeekDistance] = segmentDaysByWeeks(userMetrics?.distance, 2);
+
+            //formatted data from last week
+            let userLastWeekModeMap = {};
+            let userLastWeekSummaryMap = {};
+            if(lastWeekDistance && lastWeekDistance?.length >= 7) {
+                userLastWeekModeMap = parseDataFromMetrics(lastWeekDistance, 'user');
+                userLastWeekSummaryMap = generateSummaryFromData(userLastWeekModeMap, 'distance');
+            }
```

Comment:
Definitely never greater than 7 days, I was focused on weeks < 7 days (since that's not a week), but > days is an unexpected behavior as well, changing that to == 7 days.